tests/unit/amo/pages/TestGuides.js

```diff
@@ -0,0 +1,108 @@
+import { oneLine } from 'common-tags';
+import * as React from 'react';
+
+import NotFound from 'amo/components/ErrorPage/NotFound';
+import HeadLinks from 'amo/components/HeadLinks';
+import { fetchGuidesAddons } from 'amo/reducers/guides';
+import {
+  dispatchClientMetadata,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+import Guides, { extractId, GuidesBase, getContent } from 'amo/pages/Guides';
+
+describe(__filename, () => {
+  const getProps = ({
+    store = dispatchClientMetadata().store,
+    i18n = fakeI18n(),
+    dispatch = store.dispatch,
+    slug = 'privacy',
+    content = getContent('privacy', i18n),
+    match = {
+      params: {
+        slug,
+      },
+    },
+    ...customProps
+  } = {}) => {
+    return {
+      content,
+      dispatch,
+      i18n,
+      match,
+      slug,
+      store,
+      ...customProps,
+    };
+  };
+
+  const render = ({ ...customProps } = {}) => {
+    const allProps = getProps(customProps);
+
+    return shallowUntilTarget(<Guides {...allProps} />, GuidesBase);
+  };
+
+  it('fetches guides addons', () => {
+    const { store } = dispatchClientMetadata();
+    const dispatchSpy = sinon.spy(store, 'dispatch');
+    const root = render({ dispatch: dispatchSpy, store });
+
+    const content = getContent('privacy', fakeI18n());
+    const guids = content.sections.map((section) => section.addonGuid);
+
+    sinon.assert.calledWith(
+      dispatchSpy,
+      fetchGuidesAddons({
+        guids,
+        errorHandlerId: root.instance().props.errorHandler.id,
+      }),
+    );
+  });
+
+  it('renders a Guides Page', () => {
+    const slug = 'privacy';
+    const content = getContent(slug, fakeI18n());
+    const root = render({ content, slug });
+
+    expect(root.find('title')).toHaveText(content.title);
+
+    expect(root.find(HeadLinks)).toHaveLength(1);
+
+    expect(root.find('.Guides')).toHaveLength(1);
+    expect(root.find('.Guides-header-icon')).toHaveLength(1);
+    expect(root.find('.Guides-header-page-title')).toHaveLength(1);
+    expect(root.find('.Guides-header-intro')).toHaveLength(1);
+
+    expect(root.find('.Guides-section')).toHaveLength(1);
+    expect(root.find('.Guides-section-title').text()).toContain(
+      'Create and manage strong passwords',
+    );
+    expect(root.find('.Guides-section-description').text()).toContain(
+      'Password managers can help you',
+    );
+    expect(root.find('.Guides-section-explore-more')).toHaveHTML(
+      oneLine`<div class="Guides-section-explore-more">Explore more
+      <a href="/en-US/android/collections/mozilla/password-managers/"
+      class="Guides-section-explore-more-link">password manager</a> staff picks.</div>`,
+    );
```

Comment:
Note: you will likely have to update those assertions when you'll add more sections. Maybe you want to use `.find()` and then `.at(0)`.

Reply:
I just need to test 1 item tho, right? (`.at(0)` for example), not each section?

Reply:
yep, one section is probably fine